### PR TITLE
1566- add permissible values to InstrumentVendorEnum and InstrumentModelEnum for sequencing applications

### PR DESCRIPTION
**Pull request by aclum**
This PR expands on the two enumerations to support sequencing based vendors and models. 

We will need to write migration code to map existing values of `instrument_name` to `vendor` and `model`.

### CODE REVIEW

**Comment by aclum** [comment]
@turbomam This was a bit painful b/c you can't see the actual term IDs in the tree view. Probably would have been better to dump this and use some of the ontology tools but I'm not very familiar with those. It would be useful to have a little script that could make a permissible value out of every child of a term (in this case a good starting point would have been to make a term for all the children of OBI:0400103, then delete the ones we don't need).

File: src/schema/nmdc.yaml [hunk 1]
```diff
@@ -2118,8 +2198,16 @@ enums:
       illumina:
         aliases:
           - Illumina
-        exact_mappings:
-          - BAO:0200000
+        meaning: BAO:0200000
```
Comment (turbomam) [inline src/schema/nmdc.yaml:2201]: why are you using BAO:0200000 and not OBI:0000759 to represent illumina?

we should be suspicious of using new terms that require adding a new prefix expansion.

Comment (aclum) [reply src/schema/nmdc.yaml:2201]: @turbomam  we had a conversation about this during the Berkeley refactor. The OBI term was imported where as the BAO term wasn't so I thought the guidance at the time was to use BAO. Let me know if you want me to change this back.

**Review by cmungall — APPROVED** [review]

File: src/schema/nmdc.yaml (hunk 1 above)
Comment (aclum) [reply src/schema/nmdc.yaml]: updated to OBI term

**Review by turbomam — APPROVED** [review]

**Pull request merged**